scripts/install_minikube.sh

```diff
@@ -2,41 +2,18 @@
 set -euxo pipefail
 export SUDO=$(if [ -x "$(command -v sudo)" ]; then echo "sudo"; else echo ""; fi)
 
-function install_minikube() {
-    if [ "${DEPLOY_TARGET}" != "minikube" ]; then
-        echo "Skips installing minikube when deployment target is ${DEPLOY_TARGET}..."
-        return
-    fi
 
-    minikube_version=v1.23.2
-    if ! [ -x "$(command -v minikube)" ]; then
-        echo "Installing minikube..."
-        download_minikube $minikube_version
-        ${SUDO} mv -f ${HOME}/.arkade/bin/minikube /usr/local/bin/
-    elif [ "$(minikube version | grep version | awk -F'version: *' '{print $2}')" != "$minikube_version" ]; then
-        echo "Upgrading minikube..."
-        download_minikube $minikube_version
-        ${SUDO} mv -f ${HOME}/.arkade/bin/minikube $(command -v minikube)
-    else
-        echo "minikube is already installed and up-to-date"
-    fi
+function install_minikube() {
+    curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
```

Comment:
maybe to control the version?

Reply:
We can control the version when getting it from minikube directly, e.g.

```bash
    export MINIKUBE_VERSION=v1.20.0
    ... 
    curl -Lo minikube https://storage.googleapis.com/minikube/releases/${MINIKUBE_VERSION}/minikube-linux-amd64
```

Reply:
yes, let's control the version
it's usually better than the alternative